src/dlls/mscorpe/ceefilegenwriter.cpp

```diff
@@ -1167,6 +1167,13 @@ HRESULT CeeFileGenWriter::emitResourceSection()
         const BYTE *pbStartOfMappedMem;
         IMAGE_SECTION_HEADER *rsrc[2] = { NULL, NULL };
         S_SIZE_T cbTotalSizeOfRawData;
+
+        char *data = NULL;
+        SIZE_T            cReloc        = 0;
```

Comment:
I know you've just copied this, but since you are touching the code, could you please remove the alignment of variable names and values? We don't do it in this file and the guideline is to keep the code formatting similar to the surrounding code.